### PR TITLE
[Feature] Home page layout

### PR DESCRIPTION
**Pull request by agustinmrgo**
## Links (Jira/Trello ticket and other relevant links):
https://trello.com/c/6uZY7AKw/3-3-as-a-user-i-should-be-able-to-see-a-map-so-i-can-easily-identify-targets-in-the-future

## What & Why:
Home page layout with generic 'MainLayout' component.

## Screenshots:
0058F363-F4E8-48E6-B214-66401F4C5F47

### CODE REVIEW

File: src/locales/en.js [hunk 1]
```diff
@@ -24,6 +24,7 @@ export default {
   'logout.button': 'LOGOUT',
 
   'home.welcome': 'Welcome to React Redux Base {email}',
+  'home.current-location': 'You are here!',
```
Comment (elizabethlofredo) [inline src/locales/en.js:27]:
```suggestion
  'home.current_location': 'You are here!',
```
or camelcase but should be consistent with the other strings

File: src/pages/HomePage.js [hunk 2]
```diff
@@ -1,21 +1,54 @@
...
+const defaultCoords = { lat: -26.851311, lng: -65.702984 };
 
 const HomePage = () => {
+  const [locationStatus, setLocationStatus] = useState('');
+  const [lati, setLati] = useState(-26.851311);
+  const [long, setLong] = useState(-65.702984);
   const { user } = useSession();
 
+  useEffect(() => {
+    const success = position => {
```
Comment (elizabethlofredo) [inline src/pages/HomePage.js:18]:
```suggestion
    const success = ({ coords: { latitude, longitude }}) => {
```
You can take advantage of destructuring here

File: src/pages/HomePage.js [hunk 3]
```diff
@@ -1,21 +1,54 @@
...
+      setLati(position.coords.latitude);
+      setLong(position.coords.longitude);
+    };
+    const error = () => setLocationStatus('Unable to retrieve your location');
```
Comment (elizabethlofredo) [inline src/pages/HomePage.js:22]: This string should be on the strings file too, to be consistent

File: src/pages/HomePage.js [hunk 4]
```diff
@@ -1,21 +1,54 @@
...
+          sidebarContent={
+            <>
+              <p>
+                <FormattedMessage id="home.welcome" values={user} />
+              </p>
+              <LogoutButton />
+            </>
+          }
+          mainContent={
+            <GoogleMapReact
```
Comment (elizabethlofredo) [inline src/pages/HomePage.js:39]: For the next PR I think can be a good idea to extract all the map and coords logic to a separate component (for example "Map") and let this component just be in charge of rendering the sidebar and the map component.  WDYT?

**Review by elizabethlofredo — APPROVED** [review]
Great job!! I left some suggestions/comments 💪

File: src/pages/HomePage.js (hunk 4 above)
Comment (agustinmrgo) [reply src/pages/HomePage.js:39]: You actually read my mind! I realized about that after creating this PR, I will include that change in the next one for sure.

File: src/pages/HomePage.js [hunk 5]
```diff
@@ -1,21 +1,54 @@
...
 
 import LogoutButton from 'components/user/LogoutButton';
+import MainLayout from 'components/common/MainLayout';
+
+const defaultCoords = { lat: -26.851311, lng: -65.702984 };
 
 const HomePage = () => {
+  const [locationStatus, setLocationStatus] = useState('');
+  const [lati, setLati] = useState(-26.851311);
+  const [long, setLong] = useState(-65.702984);
```
Comment (natokra) [inline src/pages/HomePage.js:14]:
```suggestion
  const [lati, setLati] = useState(defualtCoords.lat);
  const [long, setLong] = useState(defaultCoords.lng);
```

**Review by natokra — COMMENTED** [review]
this is the only thing I found I'd change. Great job!

**Review by natokra — APPROVED** [review]

**Review by nsantos16 — APPROVED** [review]
Nothing to add to the other's comments, great job!

**Pull request merged**